man1/aur-search.1

```diff
@@ -50,7 +62,8 @@ Display output in
 format.
 .
 .TP
-.BI "\-k " key
+.BI \-D " DIR" "\fR,\fP \-\-directory=" DIR
```

Comment:
This line seems out of place? -D it not related to -k?

Reply:
Oops, copy-paste mistake.